packages/big-design/src/components/Worksheet/Modal/Modal.tsx

```diff
@@ -48,7 +57,11 @@ const InternalWorksheetModal = <T extends WorksheetItem>({ column }: WorksheetMo
   return (
     <Modal
       actions={[
-        { text: 'Cancel', variant: 'subtle', onClick: () => setOpenModal(null) },
+        {
+          text: 'Cancel',
```

Comment:
Hmmm I wonder if we should provide a way for these action texts to be overriden, this text won't be able to be localized as it is

Reply:
Should I provide a prop in the `config` object to ask for custom text and default to english?

Reply:
yeah that sounds good to me